multidimensional_sampling/README.md

```diff
@@ -0,0 +1,233 @@
+# Multidimensional Sampling
+
+This repo contains a pgen script for maestro that implements several
+multidimensional sampling methods which are driven by a simple yaml
+specification included in the env/variables section of any maestro
+specification. This yaml specification is a nested list named
+`SAMPLE_DICTIONARY`, and this repo contains several examples. 
+
+The pgen script is executed as follows
+ 
+```
+maestro run {-y} --pgen ./pgen_sample.py {sample_list.yaml},
+```
+
+where {-y} is an optional `auto-run` option, and `sample_list.yaml` is a maestro specification containing a `SAMPLE_DICTIONARY`.  
```

Comment:
I think accepted syntax for optional arguments is square brackets. `[-y]` in the command -- probably cleaner to put the in-line code line tick marks on line 15 also.

Reply:
Thanks! Please let me know if the following still needs fixing. 
```
maestro run [-y] --pgen ./pgen_sample.py {sample_list.yaml},
```

where `[-y]` is an optional `auto-run` option, and `sample_list.yaml` is a maes\
tro specification containing a `SAMPLE_DICTIONARY`.

Reply:
That looks good to me.